src/SelectBox/index.tsx

```diff
@@ -115,41 +122,41 @@ const SelectBox = (props: SelectProps) => {
     )
   }
 
-  const onFocus = (event: any) => {
+  const onFocus = (event: React.FocusEvent<HTMLSelectElement>) => {
     const { onFocus } = props
     event.preventDefault()
-    onFocus && onFocus()
+    onFocus && onFocus(event)
   }
 
-  const onBlur = (event: any) => {
+  const onBlur = (event: React.FocusEvent<HTMLSelectElement>) => {
     const { onBlur } = props
     onBlur && onBlur(event)
   }
 
-  const mobileOnChange = (e: any) => {
-    const value = props.options.find((o: any) => o.value === e.target.value)
-    props.onChange(value)
+  const mobileOnChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const option = props.options.find((o: Option) => o.value === e.target.value)
```

Comment:
(o: Option) is not needed now as Props are now properly typed :) but can serve as additional docs. However suggestion will work now so... it's up to you if you remove it or not